packages/datetime/src/dateInput.tsx

```diff
@@ -98,6 +98,12 @@ export interface IDateInputProps extends IDatePickerBaseProps, IProps {
     outOfRangeMessage?: string;
 
     /**
+     * Whether the date popover should be inline.
+     * @default true
+     */
+    popoverInline?: boolean;
```

Comment:
@cmslewis This was meant as a quick way to unblock me as I can't currently use the Date Input without the ability to make the popover not inlined.